### DISCUSSION TITLE
Running in MAC Big OS get bad interpreter error

### DISCUSSION THREAD

**Discussion by gcmanriq** [Camelcade/Perl5-IDEA#2406] [Q&A]
Hi,
I'm running Intellij Idea 2021.1.3 with the latest version of the Perl plugin available, I'm really new to Mac, when i configure perl the framework is recognizing it properly, as you can see
<img width="531" alt="2021-07-01_18-17-45" src="https://user-images.githubusercontent.com/17751295/124206693-b0248680-da98-11eb-9604-af174075e4b2.png">
But then when I press the "Run" button next to my script I get the following:
"bad interpreter: /usr/bin/perl^M: no such file or directory"
I can't identify where is being added that breakline, any idea where is set?

Thanks!

**Comment by hurricup** [comment]
Nope. Have you tried to remove it and add again?

**Reply by gcmanriq** [reply]
Yes, I'm able to setup a run configuration and execute the script like that, but the button "play" next to the code is the one not working
<img width="194" alt="2021-07-01_21-42-32" src="https://user-images.githubusercontent.com/17751295/124221815-4f577700-dab5-11eb-8348-6d57a255d0f0.png">

**Reply by hurricup** [reply]
Yes, I got it. To clarify: have you tried to remove perl from the list on the first screenshot. click apply. than add again?

**Comment by gcmanriq** [comment]
I tried, and then I also removed completely and realized that is the IDE
![2021-07-02_11-05-29](https://user-images.githubusercontent.com/17751295/124313377-a72dc680-db25-11eb-930d-90dac408e718.png)
If I remove that line then I don't have the "Run" option anymore to the left of the editor.
